cluster-autoscaler/cloudprovider/builder/builder_packet.go

```diff
@@ -21,23 +21,25 @@ package builder
 
 import (
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
-	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/packet"
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/equinixmetal"
 	"k8s.io/autoscaler/cluster-autoscaler/config"
 )
 
 // AvailableCloudProviders supported by the cloud provider builder.
 var AvailableCloudProviders = []string{
-	packet.ProviderName,
-	cloudprovider.EquinixMetalProviderName,
+	equinixmetal.ProviderName,
+	equinixmetal.EquinixMetalProviderName,
```

Comment:
This use of equinixmetal.EquinixMetalProviderName is inconsistent with line 42 below.

Did you want:
```suggestion
	cloudprovider.EquinixMetalProviderName,
```

Reply:
Yes @displague meant to use `cloudprovider.EquinixMetalProviderName`. Fixed

Reply:
@Shubham82 Yes, correct. Good find. Fixed it